### PR TITLE
Added svgpathtools as a package.

### PR DESCRIPTION
**Pull request by leycec**
Dear delightful @conda-forge/help-python team,

Please find enclosed one `svgpathtools` package, resolving upstream issue mathandy/svgpathtools#152. Shout-outs to @mathandy – the brilliant mathematician, machine-learning specialist, and Santa Cruz musician responsible for `svgpathtools` development. You rock, Andy! :guitar: 

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by leycec** [comment]
Curse you, missing license file. :facepalm:

**Comment by leycec** [comment]
**Hmm...** It looks like `svgpathtools` has [one failing test on Windows](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=343279&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=868):

```
.................F......................................................
======================================================================
FAIL: test_point_to_t (test_path.ArcTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Miniconda\conda-bld\svgpathtools_1624827965439\test_tmp\test\test_path.py", line 659, in test_point_to_t
    self.assertAlmostEqual(orig_t, computed_t, msg="arc %s at t=%f is point %s, but got %f back" % (a, orig_t, p, computed_t))
AssertionError: 0.4873706102697135 != 0.48737054659373674 within 7 places (6.367597676915082e-08 difference) : arc Arc(start=(931.141074443372+536.3734513599887j), radius=(738.0437682468166+4598.318237538378j), rotation=0.0, large_arc=False, sweep=True, end=(-543.784762354835+171.57798414651836j)) at t=0.487371 is point (193.67800869282425+4952.293895458512j), but got 0.487371 back

----------------------------------------------------------------------
```

We've [notified upstream](https://github.com/mathandy/svgpathtools/issues/152#issuecomment-869239326). With any luck, the next stable release of this package will resolve that. Since we can't reasonably resolve that from the `conda` side, we'll just ignore tests on Windows for now. `</sigh>`

**Review by dopplershift — CHANGES_REQUESTED** [review]
Since upstream has both licenses, we should include both. Also @mathandy can you confirm that you are willing to be a maintainer for this recipe?

**Comment by leycec** [comment]
@conda-forge/help-python: Everything passed! This must be a summer miracle. :sun_with_face:

**Comment by mathandy** [comment]
> @mathandy can you confirm that you are willing to be a maintainer for this recipe?
Yes, I'm willing to maintain this recipe.

File: recipes/svgpathtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    #     https://github.com/conda-forge/staged-recipes/pull/15429#issuecomment-869240403
+    - python -m unittest discover test  # [not win]
+  imports:
+    - svgpathtools
+
+about:
+  home: https://github.com/mathandy/svgpathtools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (dopplershift) [inline recipes/svgpathtools/meta.yaml:45]: I thought I posted this before, but can you try:
```suggestion
  license_file:
  - LICENSE.txt
  - LICENSE2.txt
```

Comment (leycec) [reply recipes/svgpathtools/meta.yaml:45]: **Brilliant!** I wasn't *quite* sure how to effect that and had just discovered conda/conda-build#3600 when your review dropped. I'll do exactly as you say straight-away. :+1:

**Comment by leycec** [comment]
@dopplershift: Thanks so much for the speedy Sunday evening review *and* the astute last-minute changeset. I should have known: just use a YAML list. `</facepalm>`

Also, your GitHub profile description is the funniest I've ever seen. My wife shares your academic complaints. :smile: 

@mathandy: Thanks so much for being a good sport about this!

`svgpathtools` **+** `conda` = *a better world*

File: recipes/svgpathtools/meta.yaml (hunk 1 above)
Comment (leycec) [reply recipes/svgpathtools/meta.yaml]: **Ugh.** I committed your awesome suggestion with f439f9f852ae, but GitHub doesn't seem to have picked up on that. <sup>*tnx alot, microsoft*</sup>

**Review by dopplershift — APPROVED** [review]

**Pull request merged**